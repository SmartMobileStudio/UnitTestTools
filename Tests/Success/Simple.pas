function Add(A, B: Integer): Integer;
begin
  Result := A + B;
end;