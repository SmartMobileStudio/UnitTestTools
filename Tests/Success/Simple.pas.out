SmartMS command-line compiler, (c) 2016 The Smart Company AS
Building Success\Simple.pas
[INFO] Building project ''...
[INFO] Compiling...
[INFO] Generating JavaScript...
[INFO] Linking...
